Multisig/UI/Safe Management/Safe Settings/SafeSettingsContentView.swift

```diff
@@ -12,7 +12,10 @@ struct SafeSettingsContentView: View {
 
     @ObservedObject
     var safe: Safe
-
+    
+    @State
+    var showDeleteConfirmation: Bool = false
+    
```

Comment:
and I would add the environment object here instead of accessing it through the "App.shared"